Sources/_CryptoExtras/RSA/RSA.swift

```diff
@@ -98,6 +98,14 @@ extension _RSA.Signing {
             }
         }
 
+        /// Construct an RSA public key with the specified parameters.
+        ///
+        /// Only the BoringSSL backend provides APIs to create the key from its parameters so we first create a BoringSSL
+        /// key, serialize it to PEM format, and then construct a platform specific key from the PEM representation.
+        public init<D: ContiguousBytes>(n: D, e: D) throws {
+            try self.init(pemRepresentation: BoringSSLRSAPublicKey(n: n, e: e).pemRepresentation)
```

Comment:
Let's have a backing implementation that always accepts a BoringSSL key, and then implement it twice. This avoids the encode/decode on Linux, where we don't have to pay for it.

Reply:
That’s a good idea. Alternatively we could `#if` dance in here but I didn’t want it to get out of step with the compiler conditions elsewhere. I’ll do the backing impl. Maybe that will motivate moving some of the code _back_ into BoringSSLHelpers so it can be shared by both backing implementations?

Reply:
I added the backing initializers that accept the BoringSSL key and now we don't have the conversion on BoringSSL-backed platforms.